components/Controllers/Collateral/Collateral.module.css

```diff
@@ -19,3 +19,26 @@
   text-align: center;
   margin-top: 1.5rem;
 }
+
+.stats {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+}
+
+.stats > div {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.stats > div:first-of-type,
+.stats > div:nth-of-type(3) {
+  background-color: var(--papr-5);
```

Comment:
maybe need some custom theming for this?

Reply:
I wonder if you should just use the table for this. It's not semantic, but would avoid having to replicate the theming and zebra striping in a one-off component

Reply:
used the Table component! thanks cnasc